src/components/Header.jsx

```diff
@@ -137,17 +139,15 @@ function Header() {
             {BRAND_NAME.toUpperCase()}
           </MuiLink>
         </Grid>
-        <Grid item>
-          <HeaderNavMenu links={links} />
-        </Grid>
 
-        <Grid item>
-          <IconButton onClick={(e) => setAnchorEl(e.currentTarget)}>
+        <Grid item sx={{ height: 80 }}>
```

Comment:
putting them under one `Grid item` prevents them from being spaced out like before.

Reply:
Putting them together sounds good but I'm not entirely convinced that manually setting the height here is the best idea. The height difference is caused by the different styling properties on the buttons themselves. I think the issue is the padding up top because the Login **Button** takes up 36.5px + 24px = 60.5px in height but the Avatar takes up 56 + 24 = 80px in height. So I think the fix lies in fixing the height of the AuthZoomButton.